cinema/views.py

```diff
@@ -34,6 +38,11 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _turn_params_to_int(params: list) -> list[int]:
+        if "".join(params).isalnum():
+            return [int(param) for param in params]
```

Comment:
why do you need this `if` statement ?

Reply:
> why do you need this `if` statement ?

Additional check I all `params` are int()